plugins/filters/askAI/askAI.cm

```diff
@@ -0,0 +1,14 @@
+triggers:
+  exclude:
+    branch:
+      - r/dependabot/
+
+automations:
+  generate_pr_desc:
+    if:
+      - true
```

Comment:
maybe it should be only for pr_created and on commit?

Reply:
another option is on `pr_created` and then only if you ask for re-evaluation in a comment (e.g. add comment like `/askai`)